tests/hubapi/demohub/biothing_studio/hub/databuild/mapper.py

```diff
@@ -24,39 +23,3 @@ def process(self,docs):
             else:
                 doc["has_gene"] = False
             yield doc
-
-class LineageMapper(BaseMapper):
```

Comment:
@remoteeng00 why this Mapper was removed?

Reply:
yeap, I think it should be putted back, and we left a comment on this  line about the collection name: https://github.com/newgene/biothings_docker/pull/5/files#diff-97d3455ae813c425448b4de7ca99e184dc9a00bff8ad6a1348b2cea8d53dfac2R37